api-reference/10 UI Widgets/dxGantt/1 Configuration/resourceAssignments/resourceAssignments.md

```diff
@@ -12,4 +12,183 @@ Configures resource assignments.
+
+#####See Also#####
+- [Bind to File Systems](concept/05%20Widgets/File%20Manager/10%20Bind%20to%20File%20Systems/Bind%20to%20File%20Systems.md 'Documentation/Guide/Widgets/File_Manager/Bind_to_File_Systems/')
+
+---
+
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#gantt").dxGantt({
+            resourceAssignments: {
+                dataSource: resourceAssignments,
+                keyExpr: "key",
+                resourceIdExpr: "resourceKey",
+                taskIdExpr: "taskKey"
+            },
+            //...
+        });
+    });
+
+    <!-- tab: data.js -->
+    var resourceAssignments = [{
+        'key': 0,
+        'taskKey': 3,
+        'resourceKey': 1
+    },
+    // ...
+    ];    
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-gantt ... >
+        <dxo-resource-assignments 
+            [dataSource]="resourceAssignments" 
+            keyExpr="key"
+            resourceIdExpr="resourceKey" 
+            taskIdExpr="taskKey">
+        </dxo-resource-assignments>
+        <!-- ... -->
+    </dx-gantt>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+    
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })    
+    
+    export class AppComponent {
+        resourceAssignments: ResourceAssignment[];
+        // ...
+
+        constructor(service: Service) {
+            this.resourceAssignments = service.getResourceAssignments();
+            // ...
+        }
+    }    
+    
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+    import { DxGanttModule } from 'devextreme-angular';
+    import { Service, ResourceAssignment, ... } from './app.service';
+
+    <!-- tab: app.service.ts -->
+    let resourceAssignments: ResourceAssignment[] = [{
+        'key': 0,
+        'taskKey': 3,
+        'resourceKey': 1
+    },
+    // ...   
+    ]; 
+    @Injectable()
+    export class Service {
+        getResourceAssignments(): ResourceAssignment[] {
+            return resourceAssignments;
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxGantt ... >
+            <DxResourceAssignments 
+                :data-source="resourceAssignmentsDataSource"
+                key-expr="key"
+                resource-id-expr="resourceKey"
+                task-id-expr="taskKey" />
+            <!-- ... -->
+        </DxGantt>
+    </template>
+    <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css'; 
+
+        import { 
+            DxGantt, 
+            DxResourceAssignments, 
+            //... 
+        } from 'devextreme-vue/gantt';
+        import { 
+            resourceAssignments, 
+            // ... 
```

Comment:
Use omissions wisely, because they can make the code hard to read. For example, what is omitted here or anywhere in this script?

Reply:
Users should be notified that they also have to import other Gantt objects within the import section

Reply:
To indicate that this is a place where we specify not only resource assignments and so on.